src/features/account-transaction-list.tsx

```diff
@@ -28,13 +28,13 @@ export const AccountTransactionList: React.FC<{ contractId: string }> = ({ contr
     transactionQK(TransactionQueryKeys.transactionsWithTransfersForAddressInfinite, contractId),
     ({ pageParam }) =>
       queries.fetchTransactionsWithTransfersForAddress(contractId, undefined, pageParam || 0)(),
-    { getNextPageParam, enabled: !!contractId }
+    { getNextPageParam, enabled: !!contractId, refetchOnWindowFocus: true }
```

Comment:
Are these all needed? I thought `refetchOnWindowFocus` was `true` by default

Reply:
https://github.com/hirosystems/explorer/blob/main/src/pages/_app.tsx#L43

it's set to default off here btw

Reply:
I wanted to only enable it for txs/blocks and keep it disabled for the rest or requests.